bps/bps_ApTemplate.yaml

```diff
@@ -10,38 +10,31 @@ campaign: DM-xxxxx
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
-# Memory allocated for each quantum, in MBs
+# The default BPS walltime is on the order of 3 days.
+# That's much more than the vast majority of jobs need,
+# and it can cause your jobs not to run if you're too close to a maintenance window.
+# Set the default walltime as appropriate for what you are running (12 hours below).
+site:
+  ncsapool:
+    profile:
+      condor:
+        +Walltime: 43200
+# Default memory allocated for each quantum, in MBs; can be overridden on a per-task basis.
+# The bps default is 2048 MB, the same as this example, but it's kept here as a reminder.
 requestMemory: 2048
-# CPUs to use per quantum
-requestCpus: 1
 
 # Example arguments, like the ones you would send to pipetask run from the command line
 payload:
-  runInit: true
   # UPDATE THIS to also set the output collection name
   payloadName: DM-xxxxx
   # UPDATE THIS to point to the correct repository
   butlerConfig: /repo/dc2/butler.yaml
   # UPDATE THIS and be sure it includes collections with raws, calibs, refcats, and skymaps
   inCollection: 2.2i/defaults/test-med-1
-  # Note: keep timestamp in outCollection so you don't get a zillion sub-runs
+  # UPDATE THIS output location
   output : 'u/${USER}/{payloadName}'
-  outCollection: '{output}/{timestamp}'
   # UPDATE THIS to specify what data to process
   dataQuery: "band='g' AND instrument='LSSTCam-imSim'"
 
-# Various things for bps to customize about each pipeline task
-pipetask:
-  # Initialization (optional, but a good idea; treated like its own pipeline task)
-  pipetaskInit:
-    runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --init-only --skip-existing --register-dataset-types --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
-
-# Condor backend stuff
-wmsServiceClass: lsst.ctrl.bps.wms.htcondor.htcondor_service.HTCondorService
-clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.single_quantum_clustering
-
-# Create quantum graph for use by all tasks
-createQuantumGraph: '${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -d "{dataQuery}" -b {butlerConfig} -i {inCollection} -p {pipelineYaml} -q {qgraphFile} --qgraph-dot {qgraphFile}.dot'
-
-# Main pipetask run call
-runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --extend-run --skip-init-writes --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
+# Extra options for qgraph generation; will skip existing quanta in the output collection
```

Comment:
Before this, suggest adding a memory override example for `makeWarp` in the same style as the memory override example for `imageDifference` in the ApPipe file, because it definitely needs it. For example,

```
# Various things for bps to customize about each pipeline task
pipetask:
  # Option to list other pipeline tasks being run here with default overrides
  makeWarp:
    requestMemory: 8192
```